ui/admin/tests/helpers/selectors.js

```diff
@@ -15,6 +15,7 @@ export const FIELD_DESCRIPTION = '[name=description]';
 export const FIELD_DESCRIPTION_VALUE = 'description';
 
 export const TABLE_RESOURCE_LINK = (url) => `tbody [href="${url}"]`;
+export const TABLE_LINK_SELECTOR = '.hds-table__tbody tr:first-child a';
```

Comment:
Moving `TABLE_LINK_SELECTOR` into the commonSelectors file since this const is also being used in [/acceptance/policy/read-test.js](https://github.com/hashicorp/boundary-ui/blob/f9411794c433409b25f3f28fc9eb217d34e1e87c/ui/admin/tests/acceptance/policy/read-test.js#L22), thus this will assist in code cleanup efforts.

Reply:
We should use `TABLE_RESOURCE_LINK` here. We opted to use `TABLE_RESOURCE_LINK` since we can be very specific with this selector while still being flexible. We originally used a selector like `TABLE_LINK_SELECTOR` but we had to duplicate it if we wanted to click on a resource in the second or third row. By passing in the URL and using the href, it gives us a little more flexibility.